File: Modulo07/Triangulo-v2/Retangulo.cpp
#include "Retangulo.h"

Retangulo::Retangulo(double cateto1, double cateto2, double hipotenuza) :
Triangulo(cateto1, cateto2, hipotenuza) {
}

double Retangulo::getArea() const {
    cout << "Invocando Retangulo" << endl;
    return lado1 * lado2 / 2;
}

File: Modulo09/Triangulo-CSharp/Retangulo.cs
public class Retangulo : Triangulo {
    public Retangulo(double cateto1, double cateto2, double hipotenuza) : base(cateto1, cateto2, hipotenuza) {
    }
    public override double getArea() {
        return lado1 * lado2 / 2;
    }
}
